zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -3662,55 +3657,13 @@ components:
         tenantId:
           description: The tenant ID of the incident.
           type: string
-    OperationItem:
```

Comment:
Not actually used in the spec, so I deleted them. (It's a long story why I found the ones that aren't used.....coming in another comment.)